scripts/setupPythonEnvironment.bash

```diff
@@ -4,13 +4,17 @@
 # Configuration
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 PACKAGE_DIR=$SCRIPT_DIR/../src/coreComponents/python/modules
-declare -a TARGET_PACKAGES=("geosx_mesh_tools_package"
-                            "geosx_xml_tools_package"
-                            "hdf5_wrapper_package"
-                            "pygeosx_tools_package")
+declare -a TARGET_PACKAGES=("$PACKAGE_DIR/geosx_mesh_tools_package"
+                            "$PACKAGE_DIR/geosx_xml_tools_package"
+                            "$PACKAGE_DIR/hdf5_wrapper_package"
+                            "$PACKAGE_DIR/pygeosx_tools_package"
+                            "$SCRIPT_DIR/../integratedTests/scripts/geosxats_package")
 declare -a LINK_SCRIPTS=("preprocess_xml"
                          "format_xml"
-                         "convert_abaqus")
+                         "convert_abaqus"
+                         "run_geosxats"
+                         "activate"
+                         "python")
```

Comment:
By default, this script is taking a cmake variable as an input.  If pygeosx is enabled, the version supplied will be `${CMAKE_BINARY_DIR}/lib/PYGEOSX/bin/python`, and if not it will be `${Python3_EXECUTABLE}`